.vib/mariadb/vib-verify.json

```
@@ -17,13 +17,37 @@
       ]
     },
     "verify": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/mariadb"
+        },
+        "runtime_parameters": "YXV0aDoKICBkYXRhYmFzZTogdGVzdF9kYXRhYmFzZQogIHVzZXJuYW1lOiAidXNlciIKICBwYXNzd29yZDogIkNvbXBsaWNhdGVkUGFzc3dvcmQxMjMhNCI=",
+        "target_platform": {
+          "target_platform_id": "{VIB_ENV_TARGET_PLATFORM}",
+          "size": {
+            "name": "S3"
```

Looks like a typo, AFAIF there are only `*4/*8` sizes.
```suggestion
            "name": "S4"
```

---

Indeed it is a typo. Corrected!